homepage/templates/homepage/users/sign_in.html

```diff
@@ -0,0 +1,17 @@
+{% extends "homepage/app_layout.html" %}
+
+{% block content %}      
+
+{% load crispy_forms_tags %}
+
+<div class="container py-5">
+    <h1>Sign In</h1>
+    <form method="POST">

```

Comment:
IMHO here is a good place to display errors if they are directly resultant from posting the form,
since no other place currently raises error messages to display to the user

Reply:
See my reply above